engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/WorldPreGenerationScreen.java

```diff
@@ -207,6 +211,13 @@ public void onOpened() {
         }
     }
 
+    /**
+     * Set selectedWorld when configure from WorldPreGenerationScreen
```

Comment:
Javadoc is great, but use the expanded version that covers all args/returns etc (missing javadoc here on `setNewName` - which is a poor name, you could simply flip it and call it `newNameToSet` for a better feel)